homeassistant/components/openalpr.py

```diff
@@ -0,0 +1,395 @@
+"""
+Component that will help set the openalpr for video streams.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/openalpr/
+"""
+from base64 import b64encode
+import logging
+import os
+from time import time
+
+import requests
+import voluptuous as vol
+
+from homeassistant.config import load_yaml_config_file
+from homeassistant.const import (
+    CONF_API_KEY, CONF_NAME, CONF_USERNAME, CONF_PASSWORD, ATTR_ENTITY_ID,
+    EVENT_HOMEASSISTANT_STOP)
+from homeassistant.components.ffmpeg import (
+    get_binary, run_test, CONF_INPUT, CONF_EXTRA_ARGUMENTS)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+
+DOMAIN = 'openalpr'
+DEPENDENCIES = ['ffmpeg']
+REQUIREMENTS = [
+    'git+https://github.com/pvizeli/cloudapi.git'
```

Comment:
Our requirement system will be unable to process this correctly. It will look for a valid package name, if not it will look for a valid package name by taking the text after `#`